utils/build.config

```diff
@@ -16,3 +16,4 @@ UCX=v1.13.0
 [patch_versions]
 spdk=https://github.com/spdk/spdk/commit/b0aba3fcd5aceceea530a702922153bc75664978.diff,https://github.com/spdk/spdk/commit/445a4c808badbad3942696ecf16fa60e8129a747.diff
 mercury=https://raw.githubusercontent.com/daos-stack/mercury/52b1ca7f66ab21e71e6a3cb7a993a53fff075fc7/na_ucx.patch
+ISAL_CRYPTO=https://github.com/intel/isa-l_crypto/commit/4f729766da9021e3074191bab07820a8ed6cad6b.diff
```

Comment:
Is there a corresponding PR for https://github.com/daos-stack/isa-l_crypto?

Reply:
The patch is in upstream, but there isn't a release with the patch. I.e. the patch link IS the commit in upstream:
https://github.com/intel/isa-l_crypto/commit/4f729766da9021e3074191bab07820a8ed6cad6b.diff

Reply:
Oh, I misunderstood. You mean a corresponding in _**daos-stack**_. No, there isn't. I'm assuming there should be then?

Reply:
Actually https://github.com/daos-stack/isa-l_crypto is archived